proto/api.proto

```diff
@@ -840,6 +851,15 @@ enum MarketStatus {
   MS_ONLINE = 1; // online
 }
 
+message PostSubmitMineOreResponse {
+  string status = 1;
```

Comment:
what's this for ? not sure we need a response since the miner can't do anything with it anyway

Reply:
you requested a response to the user like 200 OK